Automattic/wp-calypso
# Adds form to DIFM Lite

#### Changes proposed in this Pull Request

This change embeds the form at `/marketing/do-it-for-me/site-info/`. This is built on top of the work done in #55817.

#### Testing instructions


* The embed works only on HTTPS. You can use the calypso.live link for testing. To test locally, follow the instructions here: 2a1b7-pb
* Navigate to `/marketing/do-it-for-me/landing/?flags=difm-lite-pilot`
* Click on the "I am interested" CTA.
* Confirm that the test typeform has loaded correctly.
* Answer the survey questions.
* Confirm that submitting the survey navigates to the checkout page with the DIFM product in the cart.
* If D66148 has been applied, the cart will also contain the Premium plan product.


https://user-images.githubusercontent.com/5436027/131825271-69e3c0b8-efb9-40c2-83be-0a67dff9d040.mp4
